docs/modules/deployment/pages/core/getting-started.adoc

```diff
@@ -132,14 +133,14 @@ include::docker/initialize.adoc[]
 endif::[]
 ====
 
-== Receive SNMP Traps/Informs
+== Receive SNMP traps/informs
 
-OpenNMS {page-component-title} core lets you receive and process SNMP Traps/Informs out of the box.
-The OpenNMS {page-component-title} core services run as an unprivileged user and can't bind on port numbers < 1024 without escalated privileges.
-For this reason, the default port for the SNMP Trap/Inform listener is set to port number 10162/udp instead of the IANA registered port number 162/udp.
-The following example shows how to configure the local firewall daemon to forward port 162/udp to 10162/udp.
+OpenNMS {page-component-title} Core lets you receive and process SNMP traps/informs out of the box.
+The OpenNMS {page-component-title} Core services run as an unprivileged user and can't bind on port numbers below 1024 without escalated privileges.
```

Comment:
Is "Core" an official term we are using as a proper noun, or a term used in the docs to differentiate from Minion/Sentinel.?
```suggestion
{page-component-title} Core lets you receive and process SNMP traps/informs out of the box.
{page-component-title} services run as an unprivileged user and can't bind on port numbers below 1024 without escalated privileges.
```

Reply:
That's a good point. There is no official "Core" term, it's just to differentiate between Minion/Sentinel and the core ... it should be lower case.

Reply:
Good to know - I'll go through and correct it!